catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -914,6 +914,11 @@ public CreateResponse create(CreateStorageRequest streamCreateRequest)
         streamCreateRequest.getProperties()
                 .put(CONTENT_PATHS, tmpContentPaths);
 
+        // Get attributeOverrides, apply them and then remove them from the streamCreateRequest so they are not exposed to plugins
+        String contentDirectoryHeaders = (String)streamCreateRequest.getProperties().get("ContentDirectoryHeaders");
```

Comment:
Can we call this property something more generic that isn't tied to the content directory?  In the future, other endpoints may supply overrides in a similar way.